api/v1/views/post_boot.py

```diff
@@ -6,7 +6,7 @@
 from rest_framework.response import Response
 
 from core.models.group import Group
-from core.models.boot_script import ScriptType, BootScript,\
```

Comment:
See line 65 of this file.  CoreBootScript is not defined.  However we want the DoesNotExist exception which only exists in the model and not the view.  So the BootScript that is imported from core.models should be imported as CoreBootScript.  This is similar to the Identity model being imported as CoreIdentity found in other places in this code base.